### ISSUE TITLE
TabBar issue on iOS 15

### ISSUE THREAD

**Issue by HappyIosDeveloper** [scenee/FloatingPanel#504]
The floating panel clips out the tab bar background then I move it down!

![Simulator Screen Shot - iPhone 12 Pro Max - 2021-09-23 at 18 19 49](https://user-images.githubusercontent.com/29095193/134530600-00deb772-9df7-4d32-9576-3caa1442da67.png)

**Comment by scenee** [comment]
TabBar appearance has refined on iOS15. I confirmed the same issue in Samples app. To fix the clear background, I need to check 'Scroll Edge' of the tab bar in the storyboard or set a value to `UITabBar.scrollEdgeAppearance. As a result, the tab bar show as well as iOS 14 and below.
https://developer.apple.com/videos/play/wwdc2021/10059/
https://stackoverflow.com/questions/68688270/ios-15-uitabbarcontrollers-tabbar-background-color-turns-black

**Comment by amitpdev** [comment]
```
        let appearance = UITabBarController().tabBar.standardAppearance
        appearance.backgroundEffect = nil
        
        if #available(iOS 15.0, *) {
            tabBarController.tabBar.scrollEdgeAppearance = appearance
            tabBarController.tabBar.standardAppearance = appearance
        }
```
